cypress/integration/publishingFlows/postBrowserPersistence.spec.js

```diff
@@ -0,0 +1,285 @@
+describe('Post Editor', () => {
+  function getPostContent() {
+    return cy.get('@articleForm').findByLabelText(/^Post Content$/i);
+  }
+
+  describe('v1 Editor', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.visit('/new');
+        });
+      });
+    });
+
+    describe(`revert changes`, () => {
+      it('should revert to the initial v1 editor template if it is a new post', () => {
+        const initialContent = `---\ntitle: \npublished: false\ndescription: \ntags: \n//cover_image: https://direct_url_to_image.jpg\n---\n\n`;
+        const updatedContent = `---\ntitle: \npublished: true\ndescription: some description\ntags: tag1, tag2,tag3\n//cover_image: https://direct_url_to_image.jpg\n---\n\nThis is some text that should be reverted`;
+
+        cy.findByRole('form', { name: /^Edit post$/i }).as('articleForm');
+
+        // Fill out the title, tags, and content for an post.
+        getPostContent().clear();
+
+        // Clearing out the whole content area as this seemed simpler than finding where to add the fields in the v1 editor
+        getPostContent().click();
+        getPostContent()
+          // The v1 editor has all the post info in one textarea
+          .type(updatedContent, { force: true })
+          .should('have.value', updatedContent);
+
+        cy.findByRole('button', { name: /^Revert new changes$/i }).click();
+
+        // The post editor should reset to it's initial values
+        getPostContent().should('have.value', initialContent);
+      });
+
+      it('should revert to the previously saved version of the post if the post was previously edited', () => {
+        // Create an post and edit it.
+        const initialContent = `---\ntitle: Test Post\npublished: false\ndescription: \ntags: beginner, ruby, go\n//cover_image: https://direct_url_to_image.jpg\n---\n\nThis is a Test Post's contents.`;
+        const updatedContent = `---\ntitle: \npublished: true\ndescription: some description\ntags: tag1, tag2,tag3\n//cover_image: https://direct_url_to_image.jpg\n---\n\nThis is some text that should be reverted`;
+
+        cy.createArticle({
+          content: initialContent,
+          published: true,
+          editorVersion: 'v1',
+        }).then((response) => {
+          const { baseUrl } = Cypress.config();
+          cy.intercept(
+            baseUrl.replace(/\/$/, '') +
+              `${response.body.current_state_path.replace(
+                /\?.+$/,
+                '',
+              )}/edit?i=i`,
+          ).as('editPage');
+
+          cy.visit(response.body.current_state_path);
```

Comment:
```suggestion
          cy.visitAndWaitForUserSideEffects(response.body.current_state_path);
```

suggestion as above, to take into account any network requests that might cause issues later